### ISSUE TITLE
Distinguish sites with different protection levels by colour

### ISSUE THREAD

**Issue by bferguso** [bcgov/BCHeritage#484]
Need to distinguish 
1. LG protected
2. Provincially Protected
3. Federally Protected
sites by applying a different colour to each type.

- [ ] @KatieMcEvoy - how do we identify sites with multiple protections (LG+Provincial, etc)

@emjohnst , @KatieMcEvoy - Can we use the `Government Name` to distinguish?
`Government of Canada` = Federal, `Government of British Columbia` = Provincial, everything else Municipal? Or do we need to go to the Legislative act data? Just trying to minimize any performance issues by adding this.
- 50% opacity,
- Precedents - Provincial, Federal, Municipal
- Colour of fill and outline the same,
- Outline 100%,
- Colours as below

**Comment by bferguso** [comment]
@rajpersram - This can be included in your map L&F work.

**Comment by bferguso** [comment]
Will start implementing functionality with placeholder colours/styles

**Comment by bferguso** [comment]
Added base functionality. Federal = Green, Provincial=Blue, Municipal=Orange. 
Also added Borden Number as label on site.
@rajpersram - Please include this functionality with your map style work. Still need to decide what to do with sites that have more than one authority.

**Comment by rajpersram** [comment]
The default hex green (00FF00) will be problematic for colourblindness so we might want to avoid that to meet accessibility requirements as much as possible. The '11-class RdYlBu' colour-blind-friendly values I'll be working with (and will update this ticket with for the above designations) are attached here for starters.
![11-class RdYlBu spectrum](https://github.com/bcgov/BCHeritage/assets/18739261/046527e3-0ff4-434d-9d77-77b4701ca901)

**Comment by bferguso** [comment]
@rajpersram - Understood. The colours applied were not intended to end up in the final implementation. Didn't spend any time on them as I know (think?) you're working on an overall colour scheme. Just wanted to flag this ticket to let you know each site can have 3 (or more) colours based on the recognition type and possibly combinations of them.

**Comment by KatieMcEvoy** [comment]
![image](https://github.com/bcgov/BCHeritage/assets/123397964/8202213a-e2ad-44c7-bda0-573425fd798f)

**Comment by KatieMcEvoy** [comment]
![image](https://github.com/bcgov/BCHeritage/assets/123397964/6af20777-c216-481e-9da2-1c9d1984d213)

would this colour scheme work for differentiating the sites?

we have some ideas for combo situations - lets discuss

**Comment by rajpersram** [comment]
@KatieMcEvoy , apologies. Your comment somehow flew under my radar. Were you requesting the above colour scheme for the three different items, or were you wanting completely distinct colours? Happy to discuss. Once we land on a working solution, I should be able to make those changes myself in the app.

**Comment by KatieMcEvoy** [comment]
@rajpersram no worries. I was requesting the above scheme for the three different items.

**Comment by rajpersram** [comment]
BCRHP has been updated with the three sublayer colours indicated above. The Indigenous Reserve Boundary, which was slated for the colour (253,174,97) above is currently a grey outline without a fill. Although I'm able to adjust the fill colours myself, if they aren't stated in the JSON code already, I'll need to ask for Brett's expertise in establishing those initially.

**Comment by KatieMcEvoy** [comment]
@bferguso 
@emjohnst 

They are currently all orange.
Please refer to my comment on July 15th for colour scheme.

Addition: Could also have legacied/cancelled sites show up as either a different colour or diagonal lines of their original colour?

**Comment by bferguso** [comment]
Working version - may need to improve performance. Lets raise that as another ticket if current performance is somewhat acceptable.

**Comment by bferguso** [comment]
For now just showing single colour in order of precedence. May look into other visual coding post-mvp.

**Comment by KatieMcEvoy** [comment]
Closing, will continue to work on visual coding post mvp.
